src/main/java/com/github/tomakehurst/wiremock/recording/LoggedResponseDefinitionTransformer.java

```diff
@@ -30,6 +30,8 @@
  * Transforms a LoggedResponse into a ResponseDefinition, which will be used to construct a
  * StubMapping
  */
+@SuppressWarnings("DeprecatedIsStillUsed")
+@Deprecated(forRemoval = true)
 public class LoggedResponseDefinitionTransformer
```

Comment:
Do we need this now you reverted the change in v4.x?

Reply:
good point. should be able to take that out

Reply:
removed.